requests_ntlm/requests_ntlm.py

```diff
@@ -13,25 +13,34 @@ class HttpNtlmAuth(AuthBase):
     Supports pass-the-hash.
     """
 
-    def __init__(self, username, password, session=None):
+    def __init__(self, username, password, session=None, domain=None):
         """Create an authentication handler for NTLM over HTTP.
 
-        :param str username: Username in 'domain\\username' format
+        :param str username: Username. If in either 'domain\\\\username'
```

Comment:
Yes. I guess that's because both Python and reStructuredText treat backslash as an escape character. If HTML docs arent' generated anywhere I'm happy to just leave two backslashes. This way printing `__doc__` will display it in the correct format.